src/sections/footer.html

```diff
@@ -1,4 +1,37 @@
-<div>
-    <h1>Footer</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
+<div class="lg:hidden text-darkgreen text-[0.938rem] md:text-[1rem] font-medium">
+    <div class="md:flex justify-between md:mb-[1.313rem]"> 
+        <div id="footer__address" class="mb-[1.313rem]">
+            Agentur Baumeister <br>
+            Böckhstraße 13 <br>
+            10967 Berlin
+        </div>
+        <div id="footer__contact" class="mb-[7rem] md:mb-0 md:text-right">
+            <span class="underline">hallo@agentur-baumeister.com</span> <br>
+            +49 (0)30 - 490 827 87
+        </div>
+    </div>
+    <div id="footer__links" class="flex justify-between mb-[1.625rem] md:mb-[2.313rem]">
+        <a href="#" target="_blank">Datenschutz</a>
+        <a href="#" target="_blank">Impressum</a>
+        <a href="#" target="_blank">LinkedIn</a>
+        <a href="#" target="_blank">Instagram</a>
+    </div>
```

Comment:
Since we are basically creating a list of `anchor` elements, using a `ul` with `li` elements as subitems wrapping around the `a` elements has a more semantic meaning.

I'd suggest something like this. But there's a better way we can use JavaScript to reduce this repetition. 

Let me open it as a challenge for you guys to figure it out.🏆

```suggestion
   <ul
    id="footer__links"
    class="flex justify-between mb-7 md:mb-9"
  >
    <li>
      <a href="#" target="_blank">Datenschutz</a>
    </li>
    <li>
      <a href="#" target="_blank">Impressum</a>
    </li>
    <li>
      <a href="#" target="_blank">LinkedIn</a>
    </li>
    <li>
      <a href="#" target="_blank">Instagram</a>
    </li>
  </ul>
```